.gitignore

```diff
@@ -25,13 +25,14 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-# local env files
+# env files
 .env*.local
-.env
+.env*
 
```

Comment:
แค่ .env*  ก็พอป้ะ

Reply:
Cannot for some reason. It still .env.development still appears in the repo after adding .env*

Reply:
แต่กุได้นะมึง